[2021-06-27 18:01:45,159] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.recreate_mysql_schema 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:45,194] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.recreate_mysql_schema 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:45,197] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:45,198] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 18:01:45,202] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:45,231] {taskinstance.py:1087} INFO - Executing <Task(MySqlOperator): recreate_mysql_schema> on 2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:45,246] {standard_task_runner.py:52} INFO - Started process 7061 to run task
[2021-06-27 18:01:45,254] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'recreate_mysql_schema', '2021-06-27T18:01:27.549223+00:00', '--job-id', '85', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmpkpegi8uh', '--error-file', '/tmp/tmp24gc6ki9']
[2021-06-27 18:01:45,256] {standard_task_runner.py:77} INFO - Job 85: Subtask recreate_mysql_schema
[2021-06-27 18:01:45,316] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.recreate_mysql_schema 2021-06-27T18:01:27.549223+00:00 [running]> on host b69deda3a989
[2021-06-27 18:01:45,394] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=recreate_mysql_schema
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T18:01:27.549223+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:45,395] {mysql.py:72} INFO - Executing: -- Drop tables


DROP TABLE IF EXISTS test.invoice;

DROP TABLE IF EXISTS test.customer;

SET GLOBAL local_infile=1;
-- test.customer definition

CREATE TABLE IF NOT EXISTS `test`.`customer` (
  `id` int NOT NULL,
  `customerName` varchar(100) DEFAULT NULL,
  `address` varchar(100) DEFAULT NULL,
  `country` varchar(100) DEFAULT NULL,
  `phone` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;


-- test.invoice definition

CREATE TABLE IF NOT EXISTS `test`.`invoice` (
  `id` varchar(100) NOT NULL,
  `idcustomer` int DEFAULT NULL,
  `orderdate` date DEFAULT NULL,
  `subtotal` float DEFAULT NULL,
  `discount` float DEFAULT NULL,
  `region` varchar(100) DEFAULT NULL,
  `salesperson` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `invoice_FK` (`idcustomer`),
  CONSTRAINT `invoice_FK` FOREIGN KEY (`idcustomer`) REFERENCES `customer` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
[2021-06-27 18:01:45,422] {base.py:78} INFO - Using connection to: id: docker-mysql. Host: 172.27.240.1, Port: 3306, Schema: , Login: root, Password: ***, extra: {'charset': 'utf8', 'local_infile': True}
[2021-06-27 18:01:45,521] {dbapi.py:204} INFO - Running statement: -- Drop tables


DROP TABLE IF EXISTS test.invoice;

DROP TABLE IF EXISTS test.customer;

SET GLOBAL local_infile=1;
-- test.customer definition

CREATE TABLE IF NOT EXISTS `test`.`customer` (
  `id` int NOT NULL,
  `customerName` varchar(100) DEFAULT NULL,
  `address` varchar(100) DEFAULT NULL,
  `country` varchar(100) DEFAULT NULL,
  `phone` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;


-- test.invoice definition

CREATE TABLE IF NOT EXISTS `test`.`invoice` (
  `id` varchar(100) NOT NULL,
  `idcustomer` int DEFAULT NULL,
  `orderdate` date DEFAULT NULL,
  `subtotal` float DEFAULT NULL,
  `discount` float DEFAULT NULL,
  `region` varchar(100) DEFAULT NULL,
  `salesperson` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `invoice_FK` (`idcustomer`),
  CONSTRAINT `invoice_FK` FOREIGN KEY (`idcustomer`) REFERENCES `customer` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;, parameters: None
[2021-06-27 18:01:45,635] {dbapi.py:212} INFO - Rows affected: 0
[2021-06-27 18:01:45,790] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=recreate_mysql_schema, execution_date=20210627T180127, start_date=20210627T180145, end_date=20210627T180145
[2021-06-27 18:01:45,847] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-27 18:01:45,872] {local_task_job.py:151} INFO - Task exited with return code 0
